src/config/setup.js

```diff
@@ -30,10 +30,9 @@ function setupMeasure(dataLayer) {
   const helper = new DataLayerHelper(dataLayer, {'processNow': false});
 
   helper.registerProcessor('config',
-      (eventProcessor, eventOptions, storageProcessor, storageOptions) => {
+      (eventProcessor, eventOptions, storageProcessor, storageOptions) =>
```

Comment:
Why did you remove the braces?

Reply:
There wasn't really any reason, I put them back in so we don't have random changes in the commit history